invenio_github/utils.py

```diff
@@ -20,21 +20,21 @@
 """Various utility functions."""
 
 import json
-import yaml
-
 from datetime import datetime
 from operator import itemgetter
 
 import dateutil.parser
 import pytz
 import requests
 import six
+import yaml
```

Comment:
```suggestion
```

Reply:
this is being used for the cff file loading